draft-ietf-tls-tls13.md

```diff
@@ -2320,8 +2336,8 @@ this specification MUST negotiate TLS 1.2 or prior as specified in
 Servers MAY abort the handshake upon receiving a ClientHello with
 legacy_version 0x0304 or later.
 
-If this extension is present, servers MUST ignore the
-ClientHello.legacy_version value and MUST use only the
+If this extension is present in the ClientHello, servers MUST NOT the
```

Comment:
> servers MUST NOT the

??

Reply:
Yes.  (I had fixed some things and then failed to actually push it up.)